ts/components/screens/LoadingScreenContent.tsx

```diff
@@ -50,7 +52,11 @@ export const LoadingScreenContent = (props: LoadingScreenContentProps) => {
   });
 
   return (
-    <SafeAreaView style={styles.container}>
+    <SafeAreaView
+      style={styles.container}
+      edges={headerVisible ? ["bottom"] : undefined}
+      testID={testID}
+    >
```

Comment:
Have you double check the UI on Android too? I cannot see any Android device screen in the PR description.